applications/service_brokers/installing-template-service-broker.adoc

```diff
@@ -1,25 +1,25 @@
 [id="sb-installing-tsb"]
-= Installing the template service broker
+= Installing the {tsb-name}
 include::modules/common-attributes.adoc[]
 :context: sb-installing-tsb
 
 toc::[]
 
-You can install the template service broker to gain access to the template
+You can install the {tsb-name} to gain access to the template
 applications that it provides.
 
-IMPORTANT: The template service broker is deprecated in this release. Equivalent and better functionality is present in the Operator Framework and
+IMPORTANT: The {tsb-name} is deprecated in this release. Equivalent and better functionality is present in the Operator Framework and
```

Comment:
instead of "this release", should we say {product-title} 4.1?

Reply:
I would structure the IMPORTANT admonition as instructed by the [docs guidelines](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#admonitions). I see a few instances that can be updated in this PR.